typecats/exceptiongroups/__init__.py

```diff
@@ -0,0 +1,39 @@
+from typing import Callable, List, Optional, Tuple, Type, TypeVar, Union
```

Comment:
unless you plan to make this module a lot larger than it is, you're better off having this be plain ol' `typecats.exceptiongroups.py`. Usually it's best to leave `__init__.py` for exports and not put real code in it if it can be avoided.

Reply:
Peter! 😄 Glad to hear from you, hope you're doing well! Thank you for your feedback on this. Writing code in `__init__.py` has been a bit of a bad habit of mine which always felt wrong so thank you for the confirmation.

Reply:
:) i just happened to stumble across this and figured i'd give some unsolicted advice.